.gitignore

```diff
@@ -79,3 +79,12 @@ _ts3.4
 
 # Files copied as part of the build
 packages/types/src/generated/**/*.ts
+
+# Yarn 3
+.pnp.*
+.yarn/*
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
+!.yarn/sdks
+!.yarn/versions
```

Comment:
Can I suggest sending these in as a small PR now, before this one gets merged? That way we don't keep having 1k+ files left over when switching between this branch and others.

Reply:
Done